css/style.css

```diff
@@ -1,6 +1,94 @@
 /* General page styles */
 
 /* We should use a mobile first approach having out breack point at 768px */
+/*Comodities */
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+.carrousel-section {
+  margin: 20px;
+}
+
+.carrousel-container {
+  width: 100%;
+  margin: 0 auto;
+  box-sizing: border-box;
+}
+
+.carrousel-title {
+  width: 25px bold;
+}
+
+.carrousel-line {
+  width: 90px;
+  height: 1px;
+  margin-left: 0;
+  background-color: #63ccff;
+  border: 0;
+}
+
+.carrousel-container__carrousel {
+  padding-top: 18px;
+  width: 100%;
+  display: flex;
+  box-sizing: border-box;
+}
+
+.carrousel__img{
+  max-width: 100%;
+  height: auto;
+  width: auto;
+}
+
+.carrousel__bottom {
+  font-size: 50px;
+  width: 16%;
+  height: 24px;
+  font-weight: bold;
+  position: relative;
+}
+
+.carrousel__bottom:hover {
+  cursor: pointer;
+  transform: scale(1.2, 1.2);
+}
+
+.back {
+  margin-top: 25%;
+  left: 30px;
+  color: #fefefe;
+  position: relative;
+}
+
+.ahead {
+  margin-top: 25%;
+  right: 30px;
+  color: #fefefe;
+}
+
+.pointer {
```

Comment:
maybe this could be called `pointer-list` as just `pointer` may be confused with the mouse-over feature